mkdocs.yml

```diff
@@ -19,11 +19,12 @@ nav:
   - extending.md
   - how_it_works.md
   - Roadmap and related projects: roadmap_and_related.md
-  - API Completeness:
-    - api-completeness/index.md
-    - Supported DataFrame methods: api-completeness/dataframe.md
-    - Supporteda Expr methods: api-completeness/expr.md
-    - Supported Series methods: api-completeness/series.md
+  # Commented-out until https://github.com/narwhals-dev/narwhals/issues/1004 is addressed
+  # - API Completeness:
+  #   - api-completeness/index.md
+  #   - Supported DataFrame methods: api-completeness/dataframe.md
+  #   - Supported Expr methods: api-completeness/expr.md
+  #   - Supported Series methods: api-completeness/series.md
```

Comment:
currently there's so many `X`s it makes it look like Narwhals is really half-baked...I'd suggest removing this (temporarily) until it shows a more complete picture, hope it's ok